base/int.jl

```diff
@@ -427,12 +423,12 @@ widen(::Type{UInt64}) = UInt128
 # a few special cases,
 # Int64*UInt64 => Int128
 # |x|<=2^(k-1), |y|<=2^k-1   =>   |x*y|<=2^(2k-1)-1
-widemul(x::Signed,y::Unsigned) = widen(x)*signed(widen(y))
-widemul(x::Unsigned,y::Signed) = signed(widen(x))*widen(y)
+widemul(x::Signed,y::Unsigned) = widen(x) * signed(widen(y))
+widemul(x::Unsigned,y::Signed) = signed(widen(x)) * widen(y)
 # multplication by Bool doesn't require widening
-widemul(x::Bool,y::Bool) = x*y
-widemul(x::Bool,y::Number) = x*y
-widemul(x::Number,y::Bool) = x*y
+widemul(x::Bool,y::Bool) = x * y
+widemul(x::Bool,y::Number) = x * y
+widemul(x::Number,y::Bool) = x * y
```

Comment:
I don't know if it's worth changing at this point, but the large number of whitespace-only changes in here is annoying.